test/integration/manifestwork_test.go

```diff
@@ -0,0 +1,75 @@
+package integration
+
+import (
+	"context"
+	"fmt"
+
+	workv1 "open-cluster-management.io/api/work/v1"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+)
+
+var _ = ginkgo.Describe("ManifestWork API test", func() {
```

Comment:
these tests don't seem to cover the case of a valid manifestwork and ensuring proper permission bounding.  Where are those tests?

Reply:
yeah the e2e tests covering the actual functionalities will be placed to the work repo